pkg/storaged/filesystem/utils.jsx

```diff
@@ -176,7 +176,7 @@ export const MountPoint = ({ fstab_config, forced_options, backing_block, conten
         else
             opt_texts.push(_("stop boot on failure"));
         opt_texts = opt_texts.concat(split_options);
-        if (opt_texts.length) {
+        if (opt_texts.length && !client.in_anaconda_mode()) {
```

Comment:
I would rather see this condition a few lines above, before even the opt_texts is gettings filled, as it's irrelevant for the anaconda mode. 

Not a must.